gribi/gribi.go

```diff
@@ -79,6 +85,28 @@ func createGRIBIServer(root *oc.Root) (*server.Server, error) {
 		}
 	}
 
+	yclient, err := ygnmi.NewClient(gClient, ygnmi.WithTarget(target), ygnmi.WithRequestLogLevel(2))
+	if err != nil {
+		return nil, err
+	}
+
+	ribHookfn := func(o constants.OpType, ts int64, ni string, data ygot.ValidatedGoStruct) {
+		if o != constants.Add {
+			// TODO(wenbli): handle replace and delete :-)
+			return
+		}
+		// write gNMI notifications
+		if err := updateAft(yclient, o, ni, data); err != nil {
+			log.Errorf("invalid notifications, %v", err)
+		}
+
+		// server.WithFIBProgrammedCheck()
+		//   -> gives us a function that checks whether an ID is a tristate (ok, failed, pending)
```

Comment:
i'm not sure what this comment block is for. either add a todo or remove

Reply:
Added